network-api/networkapi/wagtailpages/pagemodels/buyersguide/homepage.py

```diff
@@ -392,25 +392,25 @@ def get_editorial_content_index(self):
         indexes = BuyersGuideEditorialContentIndexPage.objects.descendant_of(self)
         return indexes.first()
 
-    def get_hero_featured_article(self) -> Optional["BuyersGuideArticlePage"]:
+    def get_hero_featured_page(self) -> Optional["Page"]:
         try:
-            return self.hero_featured_article.localized
+            return self.hero_featured_page.specific.localized
```

Comment:
Since we are now pointing to `wagtailcore.Page`, we use wagtail's [page.specific](https://docs.wagtail.org/en/stable/reference/pages/model_reference.html#wagtail.models.Page.specific) method to return the page in its subclassed form (`BuyersGuideCampaignPage`/`BuyersGuideArticlePage`).

Not using .specific will return the page as a wagtailcore.Page model, which does not include all of the fields we need for use in the template.